README.md

```diff
@@ -81,7 +81,7 @@ namespace UltralightNet.GettingStarted
 
 			// Create View
 			// Session.DefaultSession - get default renderer session (used for saving cookies etc)
-			View view = new(renderer, 512, 512, false, Session.DefaultSession(renderer), false);
+			View view = new(renderer, 512, 512, null, Session.DefaultSession(renderer));
```

Comment:
can you make it just `View view = new(renderer, 512, 512);`?

Reply:
Yeah, I was wondering if the session was really necessary, turns out it wasn't. 😁